frontend/packages/design-system/src/components/Accordion/index.ts

```diff
@@ -0,0 +1 @@
+export { AccordionComponent as Accordion } from './Accordion.component';
```

Comment:
Not suggesting to change this, but I feel the `.component` naming pattern is less useful in a design system. Because the components are generally presentational there's less of a need for the container/component design pattern as they are always components.

Reply:
yeah fair enough. not really useful here. could still change it